src/pages/home/components/Header/Header.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from '@testing-library/react';
+
+import { instagramLink, tiktokLink, twitterLink } from './links/links';
+
+import Header from './Header';
+
+describe('Header component', () => {
+  describe('show the content on screen', () => {
+    it('renders its content', () => {
+      render(<Header />);
+
+      const octopostLogo = screen.getByAltText('octopost logo');
+
+      const tiktokIcon = screen.getByAltText('tiktok icon');
+
+      const twitterIcon = screen.getByAltText('twitter icon');
+
+      const instagramIcon = screen.getByAltText('instagram icon');
+
+      expect(octopostLogo).toBeInTheDocument();
+
+      expect(tiktokIcon).toBeInTheDocument();
+
+      expect(twitterIcon).toBeInTheDocument();
+
+      expect(instagramIcon).toBeInTheDocument();
```

Comment:
Não separe cada linha por uma linha em branco, ao inves disso use linhas em branco para separar contextos, por exemplo, o contexto de `getBy...` é um contexto diferente de `expect...`, ai podemos deixar os que são do mesmo contexto juntos, e adicinar uma linha em branco para separar os contextos

```suggestion
      const octopostLogo = screen.getByAltText('octopost logo');
      const tiktokIcon = screen.getByAltText('tiktok icon');
      const twitterIcon = screen.getByAltText('twitter icon');
      const instagramIcon = screen.getByAltText('instagram icon');

      expect(octopostLogo).toBeInTheDocument();
      expect(tiktokIcon).toBeInTheDocument();
      expect(twitterIcon).toBeInTheDocument();
      expect(instagramIcon).toBeInTheDocument();
```